packages/directory/tests/dummy/app/routes/application.js

```diff
@@ -0,0 +1,20 @@
+import { get } from '@ember/object';
+import Route from '@ember/routing/route';
+import { inject } from '@ember/service';
+import { hash } from 'rsvp';
+
+export default Route.extend({
+  user: inject(),
+
+  /**
+   * @property {Ember.Service}
+   */
+  bardMetadata: inject(),
+
+  model() {
+    return hash({
```

Comment:
Why do we need the `hash` if we are returning `undefined`?

Reply:
I copied this from the reports dummy app so I don't know for sure